src/core/libraries/companion/companion_util.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: Copyright 2025 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "common/types.h"
+
+namespace Core::Loader {
+class SymbolsResolver;
+}
+
+namespace Libraries::CompanionUtil {
+
+static constexpr uint32_t ORBIS_COMPANION_UTIL_OK = 0;
+
+static constexpr uint32_t ORBIS_COMPANION_UTIL_INVALID_ARGUMENT = 0x80000004;
```

Comment:
```suggestion
static constexpr uint32_t ORBIS_COMPANION_UTIL_INVALID_ARGUMENT = 0x80AD0004;
```
This way you can drop the bitwise AND with `0xad0000` in the return codes, since those are only used to achieve this, just in a way where the library ID is easily swappable

Reply:
error codes should go to [companion_error.h](https://github.com/shadps4-emu/shadPS4/blob/main/src/core/libraries/companion/companion_error.h)